build.sbt

```diff
@@ -28,7 +28,16 @@ val Scala212 = "2.12.18"
 val Scala213 = "2.13.12"
 val Scala3 = "3.3.1"
 ThisBuild / crossScalaVersions := Seq(Scala212, Scala213, Scala3)
-ThisBuild / scalaVersion := Scala3 // the default Scala
+ThisBuild / scalaVersion := Scala212 // the default Scala
+
+// Plugin setup stolen from Laika with love
+ThisBuild / githubWorkflowBuildMatrixExclusions ++= {
+  MatrixExclude(Map("project" -> "plugin", "java" -> JavaSpec.temurin("11").render)) ::
```

Comment:
Actually since JDK 11 _is_ your java version in the build (unlike Laika which uses Java 8), this exclude should be removed. Currently the plugin CI job is missing because of it.

Reply:
Ahhh, thank you!